### PR TITLE
Use parcel instead of webpack for quickstart.

### PR DESCRIPTION
**Pull request by StephanHoyer**
Replace webpack example with one that uses parcel. It's much simpler.

## Motivation and Context
There was an issue of a user in the gitter where (s)he wasn't able to get the webpack thing started. I recommend using parcel and it went smoothly.

## How Has This Been Tested?
yep, just followed the instructions and it worked like a charm.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation change

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated `docs/change-log.md`

### CODE REVIEW

**Comment by pygy** [comment]
:+1: Parcel is the saner option.

File: docs/installation.md [hunk 1]
```diff
@@ -52,20 +52,20 @@ import m from "mithril";
 m.render(document.body, "hello world");
 ```
 
-5. create `index.html`
+5. Create `index.html`.
 ```html
 <!DOCTYPE html>
 <body>
-    <script src="bin/app.js"></script>
```
Comment (barneycarroll) [inline docs/installation.md]: No binaries in Parcel? 🙀

**Comment by barneycarroll** [comment]
Yes, thanks @StephanHoyer 

Something was in the water when 1.0 was released – references to Webpack are all over the documentation. With bundler & module readmes it makes sense as contextual motivation (sadly with onmatch API design too) but I think as a follow up we should take the opportunity to review es6.md / jsx.md.

**Comment by panoply** [comment]
Just wanted to chime in on this one. Rollup is vasty superior to Parcel, not to mention it is faster than Parcel and far more mature. If you refer to this `[gist](https://gist.github.com/panoply/5bd3e8a5bb29715d7042ad38957e1d2f) you can get a very simple Rollup build and setup.

**Pull request merged**